### DISCUSSION TITLE
Item Prices Plugin: 'Show High Alchemy Profit' for Ironmen

### DISCUSSION THREAD

**Discussion by CubeF4rm** [runelite/runelite#15798] [Ideas]
Currently the show item prices plugin option 'show high alchemy profit' gives the profit against the grand exchange value, which is useless for Ironmen.

Could/should there be an option to show the profit measured against the 250+ gp value of a Nature Rune per-item alched? This would be assuming the user is using an infinite Fire Rune source ie: Fire Staff / Lava Staff / Tome of Fire. (Because who alchs without one of those?)

Obviously you can do the mental math on the 'show high alch value' but this would make things slightly simpler.

**Comment by Alexio17** [comment]
I thought this would've been a thing when I started my ironman a couple months ago, I totally agree.

**Comment by raiyni** [comment]
This already  exists as an issue.
